site/docs/src/json/reactor/response.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
This needs to be added to the APIs index.adoc page and the left hand nav as well.